pandas/tests/frame/test_constructors.py

```diff
@@ -165,7 +166,10 @@ def test_constructor_cast_failure(self):
         df["foo"] = np.ones((4, 2)).tolist()
 
         # this is not ok
-        msg = "Wrong number of items passed 2, placement implies 1"
+        msg = (
```

Comment:
can you use the `"|".join` pattern

Reply:
Yes, but I already did that in https://github.com/pandas-dev/pandas/pull/39991, which I would like to get merged first, and then will fix the conflics with this PR to resolve this

Reply:
thanks